src/Testing/MakesGraphQLRequestsLumen.php

```diff
@@ -30,15 +30,15 @@ trait MakesGraphQLRequestsLumen
      * @param  array  $extraParams
      * @return $this
      */
-    protected function graphQL(string $query, array $variables = null, array $extraParams = []): MakesGraphQLRequestsLumen
+    protected function graphQL(string $query, array $variables = null, array $extraParams = []): self
     {
         $params = ['query' => $query];
 
         if ($variables) {
-            $params += ['variables' => $variables];
+            $params = array_merge($params, ['variables' => $variables]);
```

Comment:
Why did you change this?

Reply:
That was unnecessary, but i don't see problems

Reply:
Let's not diverge the test classes, they should be kept as close together as possible.

Reply:
OK